pkgs/tools/networking/openconnect-sso/default.nix

```
@@ -0,0 +1,20 @@
+{ lib
+, openconnect
+, python3
+, python3Packages
+, poetry2nix
+, substituteAll
+, qt5
+}@attrs:
+
+let
+  src = builtins.fetchGit {
```

```suggestion
  src = fetchgit {
```
This is the reason for the ofborg failure.